tests/gitopsztp/talm/tests/talm_canary.go

```diff
@@ -71,4 +75,52 @@ var _ = Describe("Talm Canary Tests", Ordered, Label("talmcanary"), func() {
 		err := talmhelper.CleanupNamespace(clusterList, talmhelper.TemporaryNamespaceName)
 		Expect(err).ToNot(HaveOccurred())
 	})
+
+	Context("where all the canaries are successful", func() {
+		// 47947
+		It("should complete the CGU", polarion.ID("47947"), func() {
+
+			By("creating the cgu and associated resources", func() {
+				// cgu := talmhelper.GetCguDefinition(
```

Comment:
I think we need to create the CGU as well in order for the subsequent test steps to run. Are you planning to uncomment this part/create the CGU?

Perhaps add only the helper functions in this PR and create a separate one for adding the test once it's ready?